src/components/PopupDelete.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
+export default class PopupDelete extends Popup {
```

Comment:
`PopupDelete`

Please, name the class according to the check list: `PopupWithConfirmation` because you can confirm anything here (not only deleting)

Reply:
* You haven't corrected https://github.com/SalpStick/se_project_aroundtheus/pull/5#discussion_r1581307611